askama_shared/src/filters/mod.rs

```diff
@@ -77,6 +78,14 @@ where
     escape(i)
 }
 
+/// Casts number to f32
+pub fn float<T>(number: T) -> Result<f32>
```

Comment:
Same issue here -- why `f32`, not `f64`?

Reply:
We adjust it to f64
.